### ISSUE TITLE
Missing dependencies: dbus and PyQt4

### ISSUE THREAD

**Issue by alxbl** [GoSecure/pyrdp#82]
When trying to build inside of a virtualenv or pipenv shell, Packages dbus (pip install `python-dbus`) and `PyQt4` are missing.

```
 $ pyrdp-player.py ~/../rdp_replay_20190303_17-03-47_36.pyrdp                                                                           py-3.7.0 10s master :: 41d :: ⬢
Traceback (most recent call last):
  File "/home/alex/.bin/py-3.7.0/bin/pyrdp-player.py", line 4, in <module>
    __import__('pkg_resources').run_script('pyrdp==1.0.0', 'pyrdp-player.py')
  File "/home/alex/.bin/py-3.7.0/lib/python3.7/site-packages/pkg_resources/__init__.py", line 664, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/home/alex/.bin/py-3.7.0/lib/python3.7/site-packages/pkg_resources/__init__.py", line 1444, in run_script
    exec(code, namespace, namespace)
  File "/home/alex/.bin/py-3.7.0/lib/python3.7/site-packages/pyrdp-1.0.0-py3.7-linux-x86_64.egg/EGG-INFO/scripts/pyrdp-player.py", line 19, in <module>
    import notify2
  File "/home/alex/.bin/py-3.7.0/lib/python3.7/site-packages/notify2-0.3.1-py3.7.egg/notify2.py", line 39, in <module>
    import dbus
ModuleNotFoundError: No module named 'dbus'
```

and after manually installing `python-dbus`:

```
Traceback (most recent call last):
  File "/home/alex/.bin/py-3.7.0/bin/pyrdp-player.py", line 4, in <module>
    __import__('pkg_resources').run_script('pyrdp==1.0.0', 'pyrdp-player.py')
  File "/home/alex/.bin/py-3.7.0/lib/python3.7/site-packages/pkg_resources/__init__.py", line 664, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/home/alex/.bin/py-3.7.0/lib/python3.7/site-packages/pkg_resources/__init__.py", line 1444, in run_script
    exec(code, namespace, namespace)
  File "/home/alex/.bin/py-3.7.0/lib/python3.7/site-packages/pyrdp-1.0.0-py3.7-linux-x86_64.egg/EGG-INFO/scripts/pyrdp-player.py", line 20, in <module>
    from PyQt4.QtGui import QApplication
ModuleNotFoundError: No module named 'PyQt4'
```

DBUS is an easy fix, but PyQt4 doesn't install properly from pip and requires additional shenanigans. Both of these should be handled/ensured by the build so that pyrdp can run in a pipenv environment (I suspect this will still have hard dependencies on Qt4 libraries on the system, however, so those might need to be manually installed and mentioned in the README.)

Quick reference: http://movingthelamppost.com/blog/html/2013/07/12/installing_pyqt____because_it_s_too_good_for_pip_or_easy_install_.html

**Comment by xshill** [comment]
I'll add dbus for now.

For Qt, we could try using Qt5 to see if it can be done in a virtualenv. It's on PyPI so at least there's that: https://pypi.org/project/PyQt5/

I don't think we'd have to change much of the code to go from Qt4 to Qt5, but if it's too much trouble, we'll have to keep it like this for the time being.

**Comment by xshill** [comment]
After some research, it seems like PySide2 will be the way to go from now on since it's the port backed by the company. It seems like both PyQt5 and PySide2 have pretty much the same API though.

I've tested both PyQt5 and PySide2 in a pipenv in a fresh vm. The only problem I ran into was that there was a missing package (libxkbcommon-x11-0). Once installed, the script ran smoothly. It seems like a pretty common package (on my system, i3 depends on it). I'll try again with a more standard Ubuntu installation (the VM I used was an Ubuntu box from the Vagrant repos).
